ui/src/components/forms/UserFormContainer.js

```diff
@@ -23,6 +24,34 @@ const UserFormContainer = (props) => {
         // Form is not published, hence it is unavailable
         if (!data[0]) history.push('/form-unavailable');
         else {
+          const formAvailableFrom = DateTime.fromISO(
+            data[0].formAvailableFrom
+          ).setZone('Asia/Hong_Kong');
+          const formAvailableUntil = DateTime.fromISO(
```

Comment:
Should this be DateTime.fromISO().toLocal()?

Reply:
hmm, i tried both. both still relies on local device's time settings. so changing the date to the future will unlock the form. i think the only way to solve this is to use our server's datetime instead of client's datetime